File: 1st/Sunhee/programmers/숫자 문자열과 영단어.java
package Sunhee.programmers;


class Solution {
    public int solution(String s) {
        String[] arr = {"zero","one","two","three","four","five","six","seven","eight","nine"};
        
        for(int i=0;i<arr.length;i++) {
        	if(s.contains(arr[i])) {
        		s = s.replace(arr[i], Integer.toString(i));
        	}
        }
        return Integer.parseInt(s);
    }
}

File: 4st/Hani/programmer/숫자 문자열과 영단어.py
def solution(s):
    answer = 0
    
    # 해당 문자열과 해당 인덱스 일치 이용
    num = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"] 
    
    for i in range(len(num)) :
        s = s.replace(num[i], str(i))
        
    answer = int(s)
    return answer